src/demo/data.js

```diff
@@ -26,27 +30,112 @@ const enrollments = [...Array(numEnrollments)].map((_, idx) => {
     course_price: course.price,
     course_start: courseStart,
     course_end: courseEnd,
-    passed_timestamp: Math.random() <= hasPassed ? faker.date.between(courseStart, courseEnd)
+    passed_timestamp: hasPassed ? faker.date.recent(7)
```

Comment:
Does it make sense that the `passed_timestamp` is restricted to the past 7 days? Technically, the passed timestamp could be anywhere from course start to course end dates. I imagine you're setting it to the past 7 days such that the data contains at least some learners who have passed in the past week for the reports that filter in the past week. Would something like this make more sense?:

```javascript
const percentPassedPastWeek = 0.25;

return {
  ...
  passed_timestamp: Math.random() <= percentPassedPastWeek ? fake.date.recent(7) : faker.date.between(courseStart, courseEnd),
  ...
}
```

Reply:
I did this for the same reason so that data is consistent.